shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -3800,16 +3809,7 @@ module MakeImpl<LocationSig Location, InputSig<Location> Lang> {
 
       import BooleanCallContext
```

Comment:
This here is why it's problematic to remove the parameterisation on `LocalCc`. It's possible that we want to always use call-context sensitivity + local call context sensitivity, but if so, then that should be investigated on its own.

Reply:
I have reverted this change.